index.html

```diff
@@ -102,9 +102,124 @@ <h2>Work</h2>
                 <h2>MY RECENT WORK</h2>
             </div>
           </article>
-         
-        </div>   
-    </main>
+
+          <article class="down">
+            <div class="box-1">
+                <img src="./images/ImgPlaceholder.png" alt="Blank square">
+                <div class="box-2">
+                    <h1>Multi-Post Stories</h1>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                    <ul>

```

Comment:
- [ ] Please know that the `article` tag should only be used with repeated sections or something similar, like the cards for the projects those divs can be articles.
- [ ] Please know that you need to use a grid when styling the section of the projects and not leave it as the default block display, kindly fix this. 😁
- [ ] Please also know that `h1` tag can be used only once in a webpage, and it should only be used for the main text or title, and you already used it before in your headline, kindly change this to another `hx` tag.